packages/earth-admin/src/pages-client/users/home.tsx

```diff
@@ -0,0 +1,318 @@
+                        isValidNewOption={(value) => validEmail(value)}
+                        isMulti
+                        placeholder="Enter user emails to submit invite"
+                        styles={customStylesUsers}
+                        theme={(theme) => ({
+                          ...theme,
+                          ...SELECT_THEME,
+                        })}
+                        rules={{ required: true }}
+                      />
+                    </div>
+                    <div className="ng-width-2-12 ng-margin-bottom ng-padding-remove">
+                      <Controller
+                        as={Select}
+                        control={control}
+                        className="marapp-qa-invite-role"
+                        name="role"
+                        options={availableGroups}
+                        isSearchable={false}
+                        placeholder="Select role"
+                        styles={customStylesRoles}
+                        theme={(theme) => ({
+                          ...theme,
+                          ...SELECT_THEME,
+                        })}
+                        rules={{ required: true }}
+                      />
+                    </div>
+                    {serverErrors.length > 0 && <ErrorMessages errors={serverErrors} />}
+                    {usersFeedback.length > 0 && (
+                      <div className="ng-width-1-1 ng-margin-bottom">
+                        {usersFeedback.map((error) =>
+                          error.hasSuccess ? (
+                            <p className="ng-margin-remove">
+                              The email, {error.title} has been invited.
+                            </p>
+                          ) : (
+                            error.hasError && (
+                              <p className="ng-margin-remove">
+                                The email, {error.title} is unavailable. Please enter a new email to
+                                add users to your organization.
+                              </p>
+                            )
+                          )
+                        )}
+                      </div>
+                    )}
+                    <div className="ng-width-1-1 ng-margin-top">
+                      {isLoadingInviteUsers ? (
+                        <>
+                          <div className="ng-padding-medium ng-position-relative">
+                            <Spinner size="small" />
+                          </div>
+                        </>
+                      ) : (
+                        <>
+                          <button
+                            onClick={addUsersHandler}
+                            className="ng-button ng-button-primary ng-button-large ng-margin-medium-right marapp-qa-actionsubmit"
+                          >
+                            Add users
+                          </button>
```

Comment:
This should be disabled when there is no value in the input field, or the form is not dirty